AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -350,41 +348,56 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   return [super calculateSizeThatFits:constrainedSize];
 }
 
-#pragma mark - Private methods -- only call with lock.
+#pragma mark - Private methods, safe to call without lock
 
-- (void)_updateProgressImageBlockOnDownloaderIfNeeded
+- (void)handleProgressImage:(UIImage *)progressImage progress:(CGFloat)progress downloadIdentifier:(nullable id)downloadIdentifier
```

Comment:
I didn't document it, but my original intention here is that underscored methods should assume the lock is held. It's not *really* private since it's called into from the downloader. I'll add some kind of documentation.